public/grpc-client.js

```diff
@@ -32,23 +32,19 @@ async function getCredentials(lndSettingsDir) {
   return grpc.credentials.createSsl(lndCert);
 }
 
-function getMetadata(lndSettingsDir) {
-  const metadata = new grpc.Metadata();
-  const macaroonPath = path.join(lndSettingsDir, 'admin.macaroon');
-  const macaroonHex = fs.readFileSync(macaroonPath).toString('hex');
-  metadata.add('macaroon', macaroonHex);
-  return metadata;
+function getMacaroonCreds(lndSettingsDir) {
+  return grpc.credentials.createFromMetadataGenerator(function(args, callback) {
+    const metadata = new grpc.Metadata();
+    const macaroonPath = path.join(lndSettingsDir, 'admin.macaroon');
```

Comment:
It's worth noting that soon the macaroons will be stored in the network directory. Do we want to hold off on this PR until this happens?